docker/mainnet/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.7'
+
+services:
+  run_node:
+    image: ghcr.io/cheqd/cheqd-node:${CHEQD_MAINNET_VERSION}
+    user: cheqd
+    ports: # target -> port inside container, published -> port on the host
+      - target: 26656 # P2P Service Port inside container (please don't change this unless you know what you're doing)
+        published: ${TM_P2P_PORT}
+        protocol: tcp
+        mode: host
+      - target: 26657
+        published: ${TM_JSON_RPC_PORT}
+        protocol: tcp
+        mode: host
+      - target: 1317
+        published: ${TM_REST_API_PORT}
+        protocol: tcp
+        mode: host
+      - target: 9090
+        published: ${TM_GRPC_SERVER_PORT}
+        protocol: tcp
+        mode: host
+      - target: 9091
+        published: ${TM_GRPC_API_GATEWAY_PORT}
+        protocol: tcp
+        mode: host
+    env_file:
+      - mainnet-container.env
+    volumes:
+      - type: volume
+        source: cheqd_volume
+        target: /home/cheqd/.cheqdnode
```

Comment:
Slight change here: make this just `/home/cheqd` at the mountpoint. AFAIK all of the contents are within .cheqdnode except sometimes the keyring files. Anyway, better to make the volume target the overall folder for the cheqd user instead of a subdirectory.